js_modules/dagster-ui/packages/ui-core/src/runs/QueuedRunsBanners.tsx

```diff
@@ -13,15 +13,16 @@ import {useCanSeeConfig} from '../instance/useCanSeeConfig';
 export const QueuedRunsBanners = () => {
   const canSeeConfig = useCanSeeConfig();
 
+  if (!canSeeConfig) {
```

Comment:
This is a tiny fix to this same area. When this renders a Box with no contents, it causes it to take up space in the flex box:

Before:

<img width="428" height="250" alt="Image" src="https://github.com/user-attachments/assets/15d350c8-52c0-47ae-a43f-99aaf8701bc6" />

After:

<img width="416" height="255" alt="Image" src="https://github.com/user-attachments/assets/b47e8ba4-c6a9-47fc-931d-f1fb4e515256" />